### DISCUSSION TITLE
Simple static site app (and PHP)

### DISCUSSION THREAD

**Discussion by fu-sen** [caprover/caprover#1518] [Ideas]
When I watched this discussion, I saw him editing CapRover's nginx configuration and breaking CapRover.
When you want to publish a static site, ideally it should be a simple app.
I wondered why this wasn't in One Click Apps.
(Later I realized it was because you had to edit the file directly. One Click Apps can be completed almost entirely on the web)

https://github.com/fu-sen/CapRover-nginx

Since I'm more familiar with Caddy than nginx.

https://github.com/fu-sen/CapRover-Caddy

Of course you can also choose Apache.

https://github.com/fu-sen/CapRover-Apache

or PHP

https://github.com/fu-sen/CapRover-PHP-Apache

If you were running Apache, nginx or Caddy directly on your server, you can use this to switch to CapRover.
I thought this information would be needed by some users as well. The usage of your CapRover will expand.

**Comment by webma5ter23** [comment]
Thank you very much!! it was very helpful for me who just started with CapRover :D

**Comment by xtiang7c0** [comment]
any chance you can publish this again @fu-sen ?
ty
